packages/server/src/helpers/sanitize-address-middleware.js

```diff
@@ -0,0 +1,14 @@
+import { isAddress } from 'web3-utils'
+
+const sanitizeAddresses = object => {
+  Object.keys(object).forEach(key => {
+    if (!!object[key] && typeof object[key] === 'object') sanitizeAddresses(object[key])
+    else if(isAddress(object[key])) object[key] = object[key].toLowerCase()
+  })
+}
+
+export default () => (req, res, next) => {
+  sanitizeAddresses(req.query)
```

Comment:
Ah, this is quite a cute middleware trick! Never thought of doing it like this, but it's a good way to dedupe the logic!